static/js/login_or_signup.js

```diff
@@ -123,16 +123,24 @@ $(function() {
     attachSubmitClickHandler(signupFormMetadata, (formDataAsJsonString) => {
         $.post('/user', formDataAsJsonString, (responseBody, _, request) => {
             if (request.status === 201) {
-                alert('Successfully created user');
+                $("#signup-tab").removeClass("mui--is-active")
```

Comment:
There's a more correct way of doing this. See here: https://www.muicss.com/docs/v1/css-js/tabs
Try: `mui.tabs.activate(paneId)`

Reply:
nice find!